hvcc/generators/ir2c/static/HvUtils.h

```diff
@@ -273,6 +273,23 @@ static inline hv_int32_t __hv_utils_min_i(hv_int32_t x, hv_int32_t y) { return (
   }
 #endif
 #define hv_min_max_log2(a) __hv_utils_min_max_log2(a)
+#define hv_if_f(a, b, c) ((a) ? (b) : (c))
```

Comment:
or does this need to be a static inline?

Reply:
i am still unsure if there could be strange cases, but can't come up with any now.
Will hopefully be fine!